### PR TITLE
ST-4106: Uplift metrics related jars to CP Images

### PR DESCRIPTION
**Pull request by jhlee8**
Add disk-usage-agent

Migrate https://github.com/confluentinc/cc-docker-base/tree/master/disk-usage-agent to `common-docker` so we can add `disk-usage-agent.jar` as dependency (to be accessed from artifactory) for `common-docker/base` (and also use it for `cc-docker-base`) as part of the request in ST-4106.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ dockerfile {
     usePackages = true
     dockerRepos = ['confluentinc/cp-base-new', 'confluentinc/cp-jmxterm']
     slackChannel = '#tools-notifications'
-    mvnSkipDeploy = true
+    mvnSkipDeploy = false
```
Comment (andrewegel) [inline Jenkinsfile:9]: Whats the nature of this change?

**Comment by andrewegel** [comment]
Do we need to modify `base/Dockerfile.ubi8` to include these new libs?

File: Jenkinsfile (hunk 1 above)
Comment (jhlee8) [reply Jenkinsfile:9]: After migrating https://github.com/confluentinc/cc-docker-base/tree/master/disk-usage-agent to `common-docker`, we want to deprecate the build of `disk-usage-agent` from `cc-docker-base` and add it as a dependency that can be downloaded from Artifactory.

My understanding is that with this change, `disk-usage-agent` will be published to Artifactory...if this is not the right/best way to do this (as I am still figuring things out with maven), please let me know.

**Comment by jhlee8** [comment]
> Do we need to modify `base/Dockerfile.ubi8` to include these new libs?

No modification is needed since `disk-usage-agent` gets added to `/usr/share/java/` as part of https://github.com/confluentinc/common-docker/blob/master/base/Dockerfile.ubi8#L109.

File: Jenkinsfile (hunk 1 above)
Comment (andrewegel) [reply Jenkinsfile:9]: Forgive me, I miss read the variable and incorrectly assumed we were disabling the deploy component. LGTM then.

**Review by andrewegel — APPROVED** [review]

**Pull request merged**